symphony-bdk-core/src/test/java/com/symphony/bdk/core/OboServicesTest.java

```diff
@@ -24,19 +25,31 @@ void setUp() {
 
   @Test
   void testOboStreams() {
-    assertNotNull(oboServices.streams());
-    assertTrue(oboServices.streams() instanceof OboStreamService);
```

Comment:
Why did you removed this check? It was explicitly added to ensure that `oboServices.streams()` returns the `OboStreamService` interface and not directly `StreamService` class

Reply:
I thought this check is always true?
```
public OboStreamService streams() {
    return oboStreamService;
  }
```

Reply:
This check prevent a future developer to unintentionally return `StreamService` instead of the `OboStreamService` since https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/symphony-bdk-core/src/main/java/com/symphony/bdk/core/OboServices.java#L25 returns the actual class and not the interface.

Maybe you could add a small explanation comment.

Reply:
Now I realize that it has not been well implemented 👍

Reply:
Okay I will revert this change.